src/fal/run_scripts.py

```diff
@@ -39,6 +39,7 @@ class CurrentTest:
 @dataclass
 class ContextConfig:
     target_path: Path
+    script_path: Path
```

Comment:
Curious why we would want script path and not the directory instead? Or the `project_dir` even?

Reply:
In the data drift example, I wanted to import the anomaly detection functions to use them in the data drift script, and the only way I could do that was to add the dir in which the functions were to the `sys.path`. I brought this up at a standup and we decided that we would add this. [Here's](https://github.com/fal-ai/fal_dbt_examples/blob/283305fac31213ea1cd2dd81b9852a0fd477c9b7/fal_scripts/data_drift.py#L3) where I used it.